docs/semgrep-cloud-platform/teams.md

```diff
@@ -0,0 +1,181 @@
+---
+slug: teams
+append_help_link: true
+title: Manage access to projects
+hide_title: true
+description: Use the Teams feature to manage access to resources.
+tags:
+  - Semgrep Cloud Platform
+---
+
+# Manage access to projects
+
+Use the **Teams** feature to manage access to resources, such as findings in your projects.
+
+![The Teams tab within the Settings page](/img/access-teams.png)
+**Figure**. The **<i class="fa-solid fa-gear"></i> Settings > Access > Teams** tab displays both top-level teams and subteams.
+
+The **Teams** feature enables admins to grant or limit access to specific projects in Semgrep Cloud Platform (SCP). **Projects** are repositories or codebases you have added to SCP for scanning. You can quickly assign projects to large groups of members by assigning teams and subteams to members of your organization.
```

Comment:
```suggestion
The **Teams** feature enables admins to grant or limit access to specific projects in Semgrep Cloud Platform (SCP). **Projects** are repositories or codebases you have added to SCP for scanning. You can quickly assign projects to large groups of members by first assigning them to teams and subteams within your organization.
```

IDK if that's the greatest way of saying that, but the original sentence was a bit confusing to me